app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Post;
+
+class PostController extends Controller
+{
+    public function store(Request $request)
+    {
+
```

Comment:
Don't keep any unnecessary whitespace

Reply:
whitespace remove, installed editorconfig plugin in my atom so .editorconfig will work.